addons/l10n_sk/__manifest__.py

```diff
@@ -22,7 +22,7 @@
     
     """,
     'depends': [
-        'account',
```

Comment:
Note that for other localizations, the dependency on account was not removed.
It would be nice to keep it standardized and have similar base dependencies for all l10n modules.

Reply:
Laurent told me to remove account when adding multilang because multilang depend on account

Reply:
Yes, that's true, just pointing it out. 
Like in France, both were kept :)

Reply:
Since account is useless, i will remove it from the France localisation too, better to be consistent :smile: